test/jdk/java/util/zip/ZipFile/ZeroDate.java

```diff
@@ -86,7 +92,7 @@ private static void testDate(byte[] data, int date, LocalDateTime expected) thro
         writeU32(data, locpos + LOCTIM, date);
 
         // ensure that the archive is still readable, and the date is 1979-11-30
-        Path path = Files.createTempFile("out", ".zip");
+        Path path = Utils.createTempFile("out", ".zip");
```

Comment:
So it looks like the test library has this utility method which allows to create temporary files within the jtreg scratch directory. Perhaps we should use it then. Having said that, is there a reason why one test method in this test now uses `Files.createTempFile(...)` and this other test method uses `Utils.createTempFile(...)`?

Reply:
Yes, you are right. We don't need explicit deletion for these files by using this util method, Utils.createTempFile.